src/box/lua/slab.cc

```diff
@@ -201,8 +201,12 @@ lbox_slab_info(struct lua_State *L)
 	luaL_pushuint64(L, stats.small.used + index_stats.totals.used);
 	lua_settable(L, -3);
 
+#ifndef SMALL_MALLOC_IMPL
 	ratio = 100 * ((double) (stats.small.used + index_stats.totals.used)
 		       / (double) arena_size);
+#else
+	ratio = 100;
+#endif
```

Comment:
Nit: Should be 0, I think. Also, better patch this code so that it works fine with zero stats:

```C
ratio = 100 * ((double) (stats.small.used + index_stats.totals.used)
		       / (double)(arena_size + 1));
```

Reply:
Done.